SharedCryptoPkg/Driver/Dxe/SharedCryptoDxe.inf

```diff
@@ -0,0 +1,67 @@
+## @file
+#  This module consolidates all crypto needed for Project Mu UEFI and provides
+#  protocol for those functions. This protocol provides a subset of BaseCryptLib.
+#
+#  see Readme.md in SharedCryptoPkg
+#  There are a few flavors of support from the protocol and this is just one version.
+#  This supports all baseCrypt function
+#
+#  Copyright (c) 2019, Microsoft Corporation. All rights reserved.
+#
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+# IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+# OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+##
+
+
+[Defines]
+  INF_VERSION                    = 0x0001001B
+  PI_SPECIFICATION_VERSION       = 0x0001000A
+  BASE_NAME                      = SharedCryptoDxe
+  FILE_GUID                      = FEA01457-E381-4135-9475-C6AFD0076C61
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = SharedCryptoDxeEntry
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64 AARCH64
+#
+
+[Sources]
+  ../CommonSharedCryptoFull.c
```

Comment:
INF files should never use '..' in the [Sources] section.  This can be resolved by moving the INF files to the root module directory and put shared sources in a directory below the INF file.

This comment applies to all INF files in the PR that use '..'

Reply:
Excellent point. Should be fixed.